### ISSUE TITLE
.NET web site is down

### ISSUE THREAD

**Issue by filipnavara** [dotnet/runtime#104230]
<img width="764" alt="image" src="https://github.com/dotnet/runtime/assets/1764393/da2dfb0e-4de1-4dd3-97b9-503235099f87">

Not sure who to ping...

**Comment by ViktorHofer** [comment]
@mairaw

**Comment by akoeplinger** [comment]
Opened an IcM: https://portal.microsofticm.com/imp/v3/incidents/incident/518371664/summary

**Comment by am11** [comment]
CI is also failing:

```
ownloading 'https://dotnet.microsoft.com/download/dotnet/scripts/v1/dotnet-install.sh'
  Attempting to install 'sdk v9.0.100-preview.5.24307.3' from public_location.
/__w/1/s/.dotnet/dotnet-install.sh: line 1: !DOCTYPE: No such file or directory
/__w/1/s/.dotnet/dotnet-install.sh: line 2: html: No such file or directory
/__w/1/s/.dotnet/dotnet-install.sh: line 3: head: No such file or directory
...
```

(maybe we can catch >500 and avoid printing the response HTML in logs 😅 )

**Comment by slang25** [comment]
For the sake of others reading this, a workaround (courtesy of @martincostello) is to use the following urls while the dotnet subdomain is down:

https://raw.githubusercontent.com/dotnet/install-scripts/main/src/dotnet-install.sh
https://raw.githubusercontent.com/dotnet/install-scripts/main/src/dotnet-install.ps1

**Comment by mairaw** [comment]
Apologies for the inconvenience. The team is currently investigating the issue.

**Comment by lucaspimentel** [comment]
Hello, team. Looks like the site just came back up again. In the future, is there a status page for this site that we can follow to get any updates?

Something similar to:
- https://status.cloud.microsoft/
- https://azure.status.microsoft/
- https://portal.microsoft.com/servicestatus
- https://www.githubstatus.com/

**Comment by mairaw** [comment]
Yes, site is back up again. Thanks for your suggestion @lucaspimentel. I'll discuss that with the team, but we probably would have the same issue with that site today 😢 it makes sense for the other sites that have multiple teams and services under.

**Comment by Frogging101** [comment]
I downloaded from master branch, thank you

**Comment by filipnavara** [comment]
FWIW a short post-mortem would be nice. I appreciate that this also triggered the process of moving the dotnet-install scripts from marketing website into a blob storage.

**Comment by richlander** [comment]
https://github.com/dotnet/install-scripts/issues/490

**Comment by GeeSuth** [comment]
I'm curious about what happened, are there any disclosure about what happened?

**Comment by GeeSuth** [comment]
> I'm curious about what happened, are there any disclosure about what happened?

The website still down. 

https://dotnet.microsoft.com/get-dotnet/dotnet-core?missing_runtime=true&arch=x86&rid=win-x86&os=win10&apphost_version=8.0.5&gui=true

**Comment by vcsjones** [comment]
> The website still down.

Thanks for pointing that out. I routed that to the right folks internally.

**Comment by mairaw** [comment]
@GeeSuth can you confirm if that URL is working for you now? We're double checking all our services now.

**Comment by filipnavara** [comment]
> can you confirm if that URL is working for you now?

Works for me (accessing from Europe).

**Comment by mairaw** [comment]
Since you requested @filipnavara - here's a post-mortem of what happened today:

We moved the website to a new Azure tenant and did not have a complete checklist of tasks for the migration. We have updated the checklist, reconfigured the missing resources on the new tenant, and are evaluating moving the storage of the .NET scripts out of the dot.net website. We're double checking all services are reconfigured appropriately as systems come back online.
